README.md

```diff
@@ -20,37 +20,20 @@ Please join the discussion on the Angle Community [Discord Server](https://disco
 
 ## Introduction to Angle
 
-### 🏅 Over-Collateralized, Decentralized, and Capital-Efficient Stablecoin Protocol
+Angle is a decentralized stablecoin protocol composed of smart contracts mostly deployed on the Ethereum blockchain. 
 
-Angle is the first over-collateralized, decentralized and capital-efficient stablecoin protocol. It offers full convertibility between stable assets and collateral at oracle value: you can swap 1 of collateral against 1 of stablecoin and conversely.
+It can be used to issue stablecoins, called agTokens, that are pegged to a specific value. This means that these tokens are designed to mirror the value of the asset they are pegged to. For example, the agEUR is the EUR-pegged stablecoin issued through Angle Protocol. 
```

Comment:
There is a duplicate here: this paragraph is in duplicate with the paragraph below. We can keep it in the roadmap thing

Reply:
In fact I believe we could do similar to the page of the dev doc: https://developers.angle.money/#angle-protocol

From that we could add a paragraph for the borrowing module and mention governance like you do and which I don't in the dev doc